stock_scanner_web/views/web_templates.xml

```diff
@@ -0,0 +1,243 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+    Copyright 2016 Angel Moya <http://angelmoya.es>
+    Copyright 2016 Eficent Business and IT Consulting Services, S.L.
+    <http://www.eficent.com>
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+
+<openerp>
+    <data>
+
+        <template id="stock_scanner_web.assets_frontend" name="Website assets">
+            <meta name="viewport"
+                  content="width=device-width, initial-scale=1"/>
+            <script type="text/javascript"
+                    src="/stock_scanner_web/static/lib/js/jquery-1.11.3.min.js"/>
+            <link rel="stylesheet"
+                  href="/stock_scanner_web/static/lib/css/jquery.mobile-1.4.5.min.css"/>
+            <script type="text/javascript"
+                    src="/stock_scanner_web/static/lib/js/jquery.mobile-1.4.5.min.js"/>
+        </template>
+
+        <template id="layout" name="Main layout">&lt;!DOCTYPE html&gt;
+            <html>
+                <head>
+                    <title>Scanner Web</title>
+                    <t t-call-assets="stock_scanner_web.assets_frontend" t-js="false"/>
+                </head>
+                <body style="background-color:powderblue;">
```

Comment:
Aren't these colors what the `_CURSES_COLORS` is for? Maybe expose that to the template via controller? Or maybe I'm misunderstanding